client/apollo/css/src/Heading/HeadingCommon.scss

```
@@ -0,0 +1,43 @@
+@use "../common/breakpoints" as breakpoints;
+@use "../common/reboot";
+
+.af-heading {
+  display: grid;
+  gap: calc(4 / var(--font-size-base) * 1rem)
+    calc(12 / var(--font-size-base) * 1rem);
+
+  &__title {
+    margin: 0;
+    grid-column: var(--texts-column);
```

il faut essayer d'être homogène sur les noms des tokens
```suggestion
    grid-column: var(--heading-grid-column);
```